src/data/faq_de.json

```
@@ -1043,6 +1043,15 @@
                         "id": "other_features",
                         "indexed": false,
                         "accordion": [
+                            {
+                                "title": "Wo kann ich Neuigkeiten über die Corona-Warn-App finden?",
+                                "anchor": "news",
+                                "active": false,
+                                "textblock": [
+                                    "Die Verfügbarkeit und Details zu neuen Versionen der Corona-Warn-App werden im <a href='../../blog/' target='_blank'>Blog</a> veröffentlicht, sobald jeweils die Software an die Stores weitergegeben wurde. (Siehe auch <a href='#where_to_get' target='_blank'>Wo bekomme ich die App?</a>.) Zusätzlich erscheinen bei Bedarf Informationen zu verschiedenen Themen, wie beispielsweise zu Verhaltensempfehlungen bei einer roten Kachel.",

```

```suggestion
                                    "Die Verfügbarkeit und Details zu neuen Versionen der Corona-Warn-App werden im <a href='../../blog/' target='_blank'>Blog</a> veröffentlicht, sobald die Software in den Stores veröffentlicht wurde. (Siehe auch <a href='#where_to_get' target='_blank'>Wo bekomme ich die App?</a>.) Zusätzlich erscheinen bei Bedarf Informationen zu verschiedenen Themen, wie beispielsweise zu Verhaltensempfehlungen bei einer roten Kachel.",
```

Instead of "Software" I recommend to use "Updates". So the sentence would be:
"[...] sobald die Updates in den Stores veröffentlicht wurde [...]".

---

Thank you!